### DISCUSSION TITLE
Will graphqlRequestBaseQuery be part of graphql-codegen plugin or will be included in rtk package?

### DISCUSSION THREAD

**Discussion by nenadfilipovic** [reduxjs/redux-toolkit#1144] [General]
As title?

https://github.com/phryneas/graphql-code-generator/blob/rtk-query/packages/plugins/typescript/rtk-query/src/graphqlRequestBaseQuery.ts

**Comment by phryneas** [comment]
If they accept it, I'd rather have it on the graphql-codegen side of things. In the long run, there might also be a bsaeQuery for use with `graphql-ws` and potentially also other technologies, so I'd like to keep those extensions more "near that technology" than shipping all of them.

It will still be able to import it from there and should tree-shake correctly, even if you do not use graphql-codegen. But also, going forward when that plugin is out, we will fully recommend just using that plugin.
